packages/zowe-explorer-ftp-extension/src/ZoweExplorerAbstractFtpApi.ts

```diff
@@ -89,11 +89,14 @@ export abstract class AbstractFtpApi implements ZoweExplorerApi.ICommon {
             try {
                 sessionStatus = await this.ftpClient(this.checkedProfile());
             } catch (e) {
-                const imperativeError = new imperative.ImperativeError({
-                    msg: "Rest API failure with HTTP(S) status 401 Authentication error.",
-                    errorCode: `${imperative.RestConstants.HTTP_STATUS_401}`,
-                });
-                throw imperativeError;
+                if (e instanceof Error && e.message.includes("PASS command failed")) {
```

Comment:
Instead of having to explicitly check for the `instanceof Error`, you could...
```suggestion
                if (`${e}`.includes("PASS command failed")) {
```

That's kind of lazy and probably shouldn't be done 😋

Reply:
I initially had this, but felt that it wasn't explicit, and it also performs unnecessary text formatting as part of the `Error.toString` operation (that's invoked as part of placing it in the template expression).